src/parser/core/Events.ts

```diff
@@ -720,9 +714,11 @@ export interface SoulbindTrait {
   icon: string,
 }
 
+export type ConduitRank = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15;
+
 export interface Conduit {
```

Comment:
is it really necessary to do this? Seems to be more of an annoyance in the long run. 
We already map explicitly to those values in the relevant function, so it should be unnecessary to have a type for it beyond the basic number.

Reply:
I ended up doing that in order to avoid typescript errors when using [this map](https://github.com/JoeyBG/WoWAnalyzer/blob/chore/affliction-ts/src/parser/warlock/affliction/modules/features/Abilities.ts#L8).

I think in general, the more precise you get with typing, the more value you get from it, such as, for instance, forgetting a conduit rank at some point in the future. The explicit declaration will make it error out.

Reply:
The codebase uses arrays for effects of conduits if you look at existing implementations 😄

Reply:
Oh! I didn't see that. Any example I can steal from?? I didn't write the above code, but I can refactor it if it's a better way to do it

Reply:
Nevermind, I got it. Fix incoming :)

Reply:
Fixed by fb9e0f147c9f48b9ede1217ca7a1d7ad021a337e